tests/Tests/500__jupyterhub/test-versions.robot

```diff
@@ -57,6 +58,16 @@ Verify All Images And Spawner
     Should Be Equal As Integers    ${length}    4
     Log To Console    ${status_list}
 
+Verify There Are No Errors With Distutil Library
+    [Documentation]    Verifies that notebook can be spawned ,
+    ...    python version is greater than or equal to 3.8 and
+    ...    there are no errors in logs
+    [Tags]    Sanity
+    ...       Tier2
+    ...       ODS-586
+    Verify Python Version In All Images
```

Comment:
As we discussed yesterday in the automation meeting, python is already verified in plugin-verification.robot and probably other places  (@tarukumar and @lugi0 might know better)